hknweb/events/templates/events/event_form_base.html

```diff
@@ -0,0 +1,81 @@
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-datetimepicker/2.5.20/jquery.datetimepicker.full.min.js" integrity="sha256-FEqEelWI3WouFOo2VWP/uJfs1y8KJ++FLh2Lbqc8SJk=" crossorigin="anonymous"></script>
+
+<link rel="stylesheet" href="{% static 'events/style.css' %}">
+{% endblock %}
+
+<div class="centered">
+    <form method="POST" id="form">
+        {% csrf_token %}
+        <table style="margin: auto; text-align: center;">
+            <tbody>
+                <tr>
+                    <td style="padding-right: 5em;">
+                        {% for field in form %}
+                            {% if field.label != "Description" %}
+                                <div class="row">
+                                    <span class="left-side">
+                                        {{ field.label }}
+                                        {% if field.field.required %}
+                                            <span class="message">*</span>
+                                        {% endif %}
+                                        {{ field }}
+                                        {{ field.errors }}
+                                    </span>
+                                </div>
+                            {% endif %}
+                        {% endfor %}
+                    </td>
+                    <td>
+                        <div class="row right-side">
+                            <span>
+                                {{ form.description.label }}
+                                {% if form.description.field.required %}
+                                    <span class="message">*</span>
+                                {% endif %}
+                                {{ form.description }}
+                                {{ form.description.errors }}
+                            </span>
+                        </div>
+                    </td>
+                </tr>
+            </tbody>
+        </table>
+        
+        Note: You can bulk delete recurring events through the websites <a href="{% url 'admin:index' %}">admin</a>.
+        <br><br>
+
+        <div id="submit">
+            <input type="submit" value="Submit" />
+            {% if messages %}
+                {% for message in messages %}
+                    <br>
+                    <span class="message">{{ message }}</span>
+                {% endfor %}
+            {% endif %}
+        </div>
+    </form>
+
+    <script>
+        $(function () {
+            $("#id_start_time").datetimepicker({
+                format: 'm/d/Y g:i A',
+                formatTime: 'g:i A',
+                validateOnBlur: false,
+            });
+        });
+        $(function () {
+            $("#id_end_time").datetimepicker({
+                format: 'm/d/Y g:i A',
+                formatTime: 'g:i A',
+                validateOnBlur: false,
+            });
+        });
+    </script>
+</div>
```

Comment:
This is amazing, thank you for making the code more concise

Reply:
yeet!